packages/frontend-2/components/settings/workspaces/Billing.vue

```diff
@@ -136,6 +136,26 @@
           />
         </div>
 
+        <div class="mt-8 text-center text-foreground-2">
+          Need help?
+          <NuxtLink
+            class="text-foreground"
+            to="https://speckle.guide/workspaces/billing.html"
+            external
+            @click="() => mixpanel.track('Workspace Docs Link Clicked')"
+          >
+            <span class="hover:underline">Read the docs</span>
```

Comment:
I had to do this annoying hack because otherwise the hover underline included the space after "docs".